response_operations_ui/forms.py

```diff
@@ -9,8 +9,8 @@
 
 
 class LoginForm(FlaskForm):
-    username = StringField('Username', [InputRequired("Username is required")])
-    password = PasswordField('Password', [InputRequired("Password is required")])
+    username = StringField('Username', [InputRequired("Please enter a username")])
+    password = PasswordField('Password', [InputRequired("Please enter a password")])
```

Comment:
You may need to have a look at the acceptance tests to aline with these changes.

Reply:
Have you got an example for me?  I ran the tests and nothing failed (related to this.... the suite was failing on other non-sign-in areas)

Reply:
It may still work as it is looking for Username is required and Password is required.

Reply:
I was being blind on thursday, the tests did need updating so I've got a PR in the tests repo for it